stores/filestore.go

```diff
@@ -3254,10 +3278,18 @@ func (ms *FileMsgStore) GetSequenceFromTimestamp(timestamp int64) (uint64, error
 			return 0, err
 		}
 		seq := slice.firstSeq
-		firstMsgInSlice := ms.getMsgIndex(slice, seq)
+		firstMsgInSlice, err := ms.getMsgIndex(slice, seq)
+		if err != nil {
```

Comment:
Same as above.

Reply:
This one no, because I need firstMsgInSlice after that... unless I declare them both with `var` before that...

Reply:
.. or I use `else` statements (since the local declaration would be visible there)

Reply:
Agree with you, since its declaration.

Reply:
I decided to declare in the `if` actually, and since it is visible in `else` and not used after that.